explainaboard/analysis/feature.py

```diff
@@ -1,39 +1,21 @@
 from __future__ import annotations
 
+from collections.abc import Callable
 import copy
 from dataclasses import dataclass, field
-from typing import Any, Optional
-
-
-@dataclass
-class BucketInfo:
-    """
-    The class is used to define a dataclass for bucketing strategy
-    Args:
-        method: the bucket strategy
-        number: the number of buckets to be bucketed
-        setting: hyper-paraterms of bucketing
-    """
-
-    method: str = "bucket_attribute_specified_bucket_value"
-    number: int = 4
-    setting: Any = 1  # For different bucket_methods, the settings are diverse
-    _type: Optional[str] = None
-
-    def __post_init__(self):
-        self._type: str = self.__class__.__name__
+from typing import Optional
 
 
 def is_dataclass_dict(obj):
     """
-    this function is used to judge if obj is a dictionary with a key `_type`
+    this function is used to judge if obj is a dictionary with a key `cls_name`
```

Comment:
It looks the description does not explain the correct behavior of this function.